sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -296,9 +286,9 @@ public Mono<RuleProperties> createRule(String topicName, String subscriptionName
      *     namespace.
      * @throws HttpResponseException If the request body was invalid, the quota is exceeded, or an error occurred
      *     processing the request.
-     * @throws IllegalArgumentException if {@code topicName} or {@code ruleName} are are empty strings.
+     * @throws IllegalArgumentException if {@code topicName} or {@code ruleName} are empty strings.
```

Comment:
I think it would be helpful to undo any auto-formatted changes if they don't add to the PR. It increases the cognitive burden and makes the PR overwhelming because I have to look at every reformatted line or changed indent.